### DISCUSSION TITLE
The list of contributors to the single file of the repository is obtained

### DISCUSSION THREAD

**Discussion by wuxinheng** [community/community#41341] [Apps, API and Webhooks]
### Select Topic Area

Product Feedback

### Body

Does such an api exist

**Comment by jfullstackdev** [comment]
what API is that ? 

this is the GitHub API

https://docs.github.com/en/rest?apiVersion=2022-11-28

**Reply by wuxinheng** [reply]
I want to get the data on this address, is there an existing API?
[sample](https://github.com/AchievedOwner/TDesignBlazor/contributors-list/master/readme.md)

Or the api for this section of data on the git page
![1671076414832](https://user-images.githubusercontent.com/49859025/207768674-a8f3e1cd-1b19-4c5e-9b41-9984e264aba7.png)

What I want to achieve is this.

![1671076502501](https://user-images.githubusercontent.com/49859025/207768852-3f15d620-1487-437a-8685-7746fb07be94.png)
![1671076526534](https://user-images.githubusercontent.com/49859025/207768909-e6e84fd4-489d-4a35-8208-c1be00b257fb.png)

The contributing member here is the contributing member of the specified file in the code base

**Reply by jfullstackdev** [reply]
that documentation itself is open source, 

https://github.com/abpframework/abp/tree/dev/docs

you can raise an issue there to ask the same question if you can't find it at the `docs` folder
